src/components/auth/signIn/signInForm1.component.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import { Button } from '@kitten/ui';
+import { ValidationInput } from '@src/components/common';
+
+export interface SignInFormType {
+  email: string;
+  password: string;
+}
+
+interface ComponentProps {
+  onSubmit: (value: SignInFormType) => void;
+}
+
+export type SignInFormProps = ThemedComponentProps & ViewProps & ComponentProps;
+
+interface State {
+  email: string;
```

Comment:
Should it be stateless component for managing for "container-component" pair?

Reply:
in this case it should emit `onChange` event for every input contained in form
looks like a good idea?

Reply:
Not very sutable according the Flux architecture. Anyway let it be.